src/components/Comments/ReplyCommentItem.js

```diff
@@ -230,7 +314,7 @@ const ReplyCommentItem = ({
             )}
           </ButtonHightlight>
         </TouchableOpacity>
-        <Text style={styles.vote(voteStyle())}>{totalVote}</Text>
+        {totalVote !== 0 && <Text style={styles.vote(voteStyle())}>{totalVote}</Text>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The condition for rendering the total vote count has been changed. This is a good practice as it avoids rendering unnecessary elements when there are no votes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->